src/services/services.ts

```diff
@@ -3839,7 +3837,9 @@ module ts {
                 if (symbolKind) {
                     pushTypePart(symbolKind);
                     displayParts.push(spacePart());
-                    addFullSymbolName(symbol);
+                    if (!symbol.isAnonymous) {
+                        addFullSymbolName(symbol);
+                    }

```

Comment:
You only got the `symbol`?

And the `symbol.declarations[0]` node is not always defined there and I know if I add that check it will breaks some other parts. 

Also as I recall it the symbol name for a default function is just default. I can't just check for `default` symbol name because a function named default will not have a name then.